Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,34 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+
+
+//Step-1: Map the tasks to durations in hours.
+const workHour = tasks.map(myElement =>
+  (Math.round(100 * (myElement.duration / 60))) / 100);
+//60 is used to show duration in hour.
+//100 is used to round with two decimals such as 0.33 or 1.23
+
+
+//Step-2: Filter out everything that took less than two hours
+//(i.e., remove from the collection)
+const morethan2Hours = workHour.filter(hour => hour >= 2);
+// durations which are less than 2, are gone.
+
+
+//Step-3: Multiply the each duration by a per-hour rate for billing
+//(you can decide yourself what Maartje should earn per hour) 
+//and sum it all up.
+const totalHour = morethan2Hours.reduce((per, total) => total += per);
+
+const earningAmount = `€ ${Math.round((totalHour * 24.5) * 100) / 100}`;
```

Comment:
So far you are doing very well, But here became the confusion.
when ever you have a hard coded numbers means something you should at least put a comment says what these numbers are!
or you may put each in a variable making them easier to understand like: 
```javascript
const SALARY = 24.5
const earningAmount = `€ ${Math.round((totalHour * SALARY) * 100) / 100}`; 
```
Actully I didn't know what the other are but i think they are useful :nerd_face:  :100:

Reply:
you are absolutely right, the numbers are meaningful in my mind but it might confuse. After that, I will define new variables for each information. Thank you :)